lib/services.js

```diff
@@ -204,6 +205,11 @@ class Services {
       _.set(global, 'LabShare.IO', this.io())
     }
 
+    if(!_.get(global, 'LabShare.Notifications')) {
+      const notificationSettings = _.get(this._config, 'notifications');
```

Comment:
the configuration should look for the property `services.notifications`

Reply:
this is what I have in my config.json in facility:
"services": {
    "listen": {
      "port": 8000,
      "url": "http://localhost"
    },
    "notifications": {
      "sms": {
        "settings": {
          "accountSid": "",
          "authToken": "",
          "defaultTelephoneNumber": ""
        }
      }
    }
  }.

this.config() already looks for 'services'